tox.ini

```diff
@@ -30,13 +38,11 @@ deps =
     pre-commit
 commands = pre-commit run --all-files
 
-
 [testenv:licenses]
-deps =
-    liccheck
+allowlist_externals =
```

Comment:
This will also need `poetry` to perform the `commands_pre` step, you can add it with `{[testenv]allowlist_externals}`.